src/components/Reports/TotalReport/TotalPeopleReport.jsx

```diff
@@ -75,35 +78,27 @@ const TotalPeopleReport = props => {
     } else if (timeRange === 'year') {
       range = 4;
     } else {
+      // eslint-disable-next-line no-console
       console.log('The time range should be month or year.');
     }
     return objectArray.reduce((acc, obj) => {
-      const key = obj['date'].substring(0, range);
+      const key = obj.date.substring(0, range);
       const month = acc[key] || [];
       month.push(obj);
       acc[key] = month;
       return acc;
     }, {});
   };
 
-  const summaryOfTimeRange = timeRange => {
-    const groupedEntries = Object.entries(groupByTimeRange(allTimeEntries, timeRange));
-    let summaryOfTime = [];
-    groupedEntries.forEach(element => {
-      const groupedUsersOfTime = Object.values(sumByUser(element[1], 'userId'));
-      const contributedUsersOfTime = filterTenHourUser(groupedUsersOfTime);
-      summaryOfTime.push({ timeRange: element[0], usersOfTime: contributedUsersOfTime });
-    });
-    return summaryOfTime;
-  };
+  // const { userProfiles } = props;
```

Comment:
We don't allow codes in comments, just remove it.

Reply:
fixed